draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -580,6 +580,81 @@ To ensure readability and replicability, the examples in this specification do
 not contain decoy digests unless explicitly stated. For an example
 with decoy digests, see (#example-simple_structured).
 
+### Recursive Disclosures {#recursive_disclosures}
+
+The algorithms above are compatible with "recursive disclosures", in which one
+selectively disclosed field reveals the existence of more selectively
+disclosable fields.  For example, consider the following JSON structure:
+
+```json
+{
+    "family_name": "Möbius",
+    "nationalities": ["DE", "FR", "UK"]
+}
+```
+
+When the Holder has multiple nationalities, the issuer may wish to conceal
+presence of any statement regarding nationalities while also allowing the
+holder to reveal each of those nationalities individually.
+This can be accomplished by first making the entries within the `nationalities`
+array selectively disclosable, and then the whole "nationalities" field.
+
+``` ascii-art
+{
+    "family_name": "Möbius",
+    "nationalities": [
+        { "...": "PmnlrRjhLcwf8zTDdK15HVGwHtPYjddvD362WjBLwro" }
+        { "...": "r823HFN6Ba_lpSANYtXqqCBAH-TsQlIzfOK0lRAFLCM" },
+        { "...": "nP5GYjwhFm6ESlAeC4NCaIliW4tz0hTrUeoJB3lb5TA" }
+    ]
+}
+
+Content of Disclosures:
+PmnlrRj... = ["16_mAd0GiwaZokU26_0i0h","DE"]
+r823HFN... = ["fn9fN0rD-fFs2n303ZI-0c","FR"]
+nP5GYjw... = ["YIKesqOkXXNzMQtsX_-_lw","UK"]
+```
+
+``` ascii-art
```

Comment:
don't we need a bit more explanation before this example..?

Reply:
probably, yeah

Reply:
https://github.com/oauth-wg/oauth-selective-disclosure-jwt/pull/431/files#r1662801181 and https://github.com/oauth-wg/oauth-selective-disclosure-jwt/pull/431/files#r1662802244 ?

Reply:
now committed to this request to be committed to the main branch ad0d13711adc080f5cc0f8cabd6b836c27ec291a and 732162fa3a403bb9129a882c613e015aa0557c56